test/unit/lib/esQueryBuilder.js

```diff
@@ -1,20 +1,45 @@
+const util = require('util');
 const chai = require('chai');
 const esQueryBuilder = require('../../../app/lib/esQueryBuilder');
 
 const expect = chai.expect;
 
+function findKeyValuePair(obj, searchKey, searchValue) {
+  return Object.keys(obj).some((key) => {
+    const value = obj[key];
+
+    if (typeof value === 'object') {
+      if (findKeyValuePair(value, searchKey, searchValue)) {
+        return true;
+      }
+    }
+
+    if (key === searchKey && value === searchValue) {
+      return true;
+    }
+
+    return false;
+  });
+}
+
 describe('esQueryBuilder', () => {
   it('should return an object', () => {
     const query = esQueryBuilder.build();
 
     expect(query).to.be.an('object');
   });
 
-  it('should return the search term as q param', () => {
+  it('should populate the query with the search term', () => {
     const searchTerm = 'search for this';
     const query = esQueryBuilder.build(searchTerm);
 
-    expect(query.body.query.multi_match.query).to.be.equal(searchTerm);
+    expect(
```

Comment:
`expect` being split over 3 lines is unusual given the character count isn't 80+
I know there are no rules in eslint to highlight this, however, in the spirit of keeping with the convention in the rest of the project it would be good to have this be
`expect(findKeyValuePair(query, 'query', searchTerm))`